components/locid/src/extensions/transform/value.rs

```diff
@@ -15,20 +15,6 @@ use tinystr::TinyAsciiStr;
 /// separated by `-`.
 /// Each part of the sequence has to be no shorter than three characters and no
 /// longer than 8.
-///
-///
-/// # Examples
-///
-/// ```
-/// use icu::locid::extensions::transform::Value;
-///
-/// let value1: Value = "hybrid".parse().expect("Failed to parse a Value.");
```

Comment:
Nit: I think it's still useful to assert that the parsing succeeded.

Suggestion: Make the test be two strings that succeed to parse, and one string that fails to parse.